test/signs_ui/alerts/state_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule SignsUi.Alerts.StateTest do
+  use ExUnit.Case, async: true
+  use SignsUiWeb.ChannelCase
+
+  describe "start_link" do
+    test "GenServer runs without crashing" do
+      @endpoint.subscribe("signs:all")
+
+      {:ok, _pid} = SignsUi.Alerts.State.start_link(interval_ms: 50)
+      Process.sleep(70)
+
+      assert_broadcast("new_alert_state", %{})
+    end
+  end
+
+  describe "handle_info" do
+    test "handles unknown messages without crashing" do
+      {:ok, pid} = SignsUi.Alerts.State.start_link()
+      send(pid, :unknown!)
+      Process.sleep(50)
```

Comment:
Rather than sleeping, since we have a reference to the process, we could use `Process.monitor` and `refute_receive` on the DOWN message.